graphql2/schema.graphql

```diff
@@ -37,6 +37,8 @@ type Query {
   # Returns a paginated list of rotations.
   rotations(input: RotationSearchOptions): RotationConnection!
 
+  upcomingHandoffTimes(input: UpcomingHandoffTimesInput): [ISOTimestamp!]!
```

Comment:
This is rotation specific, maybe `nextRotationHandoffTimes` or something to that regard to match the existing `Rotation.nextHandoffTimes`?

Reply:
A reason for keeping them similar is that they are the exact same data, one a property of the rotation with params pre-filled and the other top-level.

I'd rather be consistent with the adjective across the UI than use "next" in some places and "upcoming" elsewhere.

Another approach could be to call this one `calcRotationHandoffTimes` and take an optional `from` parameter (timestamp) that defaults to the backend clock. It might be a good idea anyhow so that the times are always consistent with the user's machine.